taskcluster/ci/android-build/kind.yml

```diff
@@ -25,7 +27,7 @@ jobs:
         # to be able to access `rustup` and `rustc` from within the cc script
         # and the gradle command. Another options could be to set those env vars
         # here like: [export, 'PATH=$HOME/.cargo/bin:$PATH'
-        - [source, taskcluster/scripts/rust/rustup-setup.sh]
+        - [source, taskcluster/scripts/rustup-setup.sh]
```

Comment:
Maybe at some point I should have more granular folder too in app-services, good call 👍